resources/views/cvs/create.blade.php

```diff
@@ -549,26 +552,82 @@
             $.ajaxSetup({
                 headers: {
                     'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
-                    // Cho em hỏi là em vào trang này cứ bị lỗi CSRF token not found: https://laravel.com/docs/csrf#csrf-x-csrf-token
-                    // Mặc dù em đã có thẻ <meta name="csrf-token" content="{{ csrf_token() }}"> ở trang lay out
                 }
             });
 
             $(".btn-submit").click(function(e) {
+
+                let skills = [];
+                $( '.donut ').each(function () {
+                    let skill = {
+                        percent :  $( this ).children("text").text(),
+                        skill_id :  $( this ).next().text(),
```

Comment:
`$( this ).next().text(),` `$( this ).children("text").text(),` có thể viết đơn giản hơn k